### DISCUSSION TITLE
Adding accelerate functionality to Serverless Framework

### DISCUSSION THREAD

**Discussion by KillDozerX2** [serverless/serverless#11725] [Ideas]
SAM has had the accelerate for some time now and people like it, serverless framework has supported deploying individual functions for some time as well. Why can't we add something like chokidar and implement a similar feature?
I'm happy to raise a PR as I've got an idea in mind but want to understand what other people think first.

**Comment by nuc-ronniehyslop** [comment]
@KillDozerX2 this seems like a good idea, I wasn't aware of SAM accelerate before this. Sometimes it can be really painful doing changes and deploying for things that can only be tested in the cloud, made especially worse with VPC bound functions...
